features/organizer/update.feature

```diff
@@ -225,3 +225,44 @@ Feature: Test updating organizers via complete overwrite
       "url": []
     }
     """
+
+  Scenario: Creating a new educational description
```

Comment:
I would move this to a dedicated file like we did for the normal description, or store in the description feature.